packages/ns-build/server.ts

```
@@ -15,12 +15,30 @@ const useCustomServerSidePropsHandler = (path: string) =>
-  event.path = event.rawPath
-  event.headers.host = event.headers['x-forwarded-host']
-  event.headers.referer =
-    event.headers['x-forwarded-proto'] +
-    '://' +
-    event.headers['x-forwarded-host']
+  event.path = event.rawPath || event.path
+  event.rawPath = event.path
+  event.headers = event.headers || {}
+  
+  let host = event.headers['x-forwarded-host'] || event.headers.host
+  if (event.multiValueHeaders && event.multiValueHeaders['x-forwarded-host'] && event.multiValueHeaders['x-forwarded-host'].length > 0) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)